app/components/infinite-list.hbs

```diff
@@ -1,15 +1,15 @@
 <div
-  class="{{@class}} overflow-y-scroll au-u-flex au-u-flex--column"
```

Comment:
I think this breaks the "auto load more items" functionality since the div doesn't scroll anymore. Not sure if it's simple to revert while keeping the styling improvements.

I think there are better ways to implement this than using a scroll event though so it might be worth simply redoing it in a seperate PR after merging this and consider it a known issue here?

Reply:
Missed the auto scroll. Should be fixed now.